tests/frontier/opcodes/test_calldataload.py

```diff
@@ -0,0 +1,140 @@
+"""
+abstract: Test `CALLDATALOAD`
+    Test the `CALLDATALOAD` opcodes.
+
+"""
+
+import pytest
+
+from ethereum_test_forks import Frontier, Homestead
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Bytecode,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+from ethereum_test_tools import Opcodes as Op
+
+
+@pytest.mark.parametrize(
+    "sstore_test_opcode,calldata_opcode,validate_length,expected_storage",
+    [
+        (
+            Bytecode(Op.SSTORE(key=0x0, value=Op.CALLDATALOAD(offset=0x0))),
```

Comment:
Looks like the code remains unchanged. It can be just parametrization by offset variable